src/isar/config/settings.py

```diff
@@ -192,18 +192,21 @@ class Settings(BaseSettings):
     # Data classification
     DATA_CLASSIFICATION: str = Field(default="internal")
 
-    # Type of missions the robot performs
-    MISSION_TYPE: str = Field(default="inspection")
+    # List of MQTT Topics
 
-    ISAR_STATE: str = Field(default="isar_state")
+    ISAR_STATE: str = Field(default="state")
 
-    ISAR_MISSION: str = Field(default="isar_mission")
+    ISAR_MISSION: str = Field(default="mission")
 
-    ISAR_STEP_STATUS: str = Field(default="isar_step_status")
+    ISAR_TASK: str = Field(default="task")
 
-    @validator("ISAR_STATE", "ISAR_MISSION", "ISAR_STEP_STATUS", pre=True, always=True)
+    ISAR_STEP: str = Field(default="step")
+
+    @validator(
+        "ISAR_STATE", "ISAR_MISSION", "ISAR_TASK", "ISAR_STEP", pre=True, always=True
+    )
     def prefix_isar_topics(cls, v, values):
-        return f"{values['ROBOT_ID']}/{v}"
+        return f"isar/{values['ROBOT_ID']}/{v}"
```

Comment:
I really like the implementation of the prefixed isar to the topics!
Maybe we should change the naming of these config variables to make it clear they are MQTT topics?
Suggestion:
`ISAR_TASK` => `TOPIC_ISAR_TASK`